authorization-common/src/main/java/com/sequenceiq/authorization/service/UtilAuthorizationService.java

```diff
@@ -92,6 +114,35 @@ public CheckResourceRightsV4Response getResourceRightsResult(CheckResourceRights
         return generateResponse(resourceRightsChecks, rightCheckResultMap);
     }
 
+    public CheckRightOnResourcesV4Response getRightOnResourcesResult(CheckRightOnResourcesV4Request request) {
```

Comment:
same here, worth to split into at least 2 methods

Reply:
Will do.